source/extensions/filters/http/jwt_authn/filter.cc

```diff
@@ -38,6 +38,23 @@ std::string generateRcDetails(absl::string_view error_msg) {
                       absl::StrJoin(absl::StrSplit(error_msg, ' '), "_"), "}");
 }
 
+std::string buildUri(const Http::RequestHeaderMap& request_headers,
```

Comment:
buildOriginalUri().   This could be move to source/common/http/utility.  other modules may need this